### ISSUE TITLE
s/organisation/organization in quickstart wizard (and elsewhere, if relevant)

### ISSUE THREAD

**Issue by mishaone** [openshiftio/openshift.io#1322]
Red Hat products and docs all use the en-US spelling, so we would always write organization instead of organisation. I would recommend changing organization in the Quickstart wizard to reflect this (and if this is used as a field anywhere else in OSIO.

![organization](https://user-images.githubusercontent.com/926572/32644908-41bb4592-c630-11e7-8013-6ad658d28fb9.png)

**Comment by mishaone** [comment]
Hi folks,

I can't see this change in the identified wizard (the quickstart menu). See screenshot:

![organisation](https://user-images.githubusercontent.com/926572/33600216-52c113f8-d9f4-11e7-9d6d-199ab051ee23.png)

Is this a change that hasn't been published yet or is this not fixed (and this issue should be reopened)? (cc: @c-koehler)

**Comment by corinnekrych** [comment]
In prod we got:
```
$ curl https://forge.api.openshift.io/forge/version
{"backendVersion":"1.0.99","forgeVersion":"3.6.1.Final"}
```
v1.0.99 of the backend, checking here: 
https://github.com/fabric8io/generator-backend/commit/5f16351379faa05021ea758d84ec6c7fef9f0668 and then the pom file associated to this backedn:
https://github.com/fabric8io/generator-backend/blob/5f16351379faa05021ea758d84ec6c7fef9f0668/pom.xml#L22 we got `1.0.74` deployed on PROD

However @c-koehler change is in https://github.com/fabric8io/fabric8-generator/commit/21657e5020a92e6d68b9f14594eb047abedc1103 ie: `1.0.76` which is packaged in `v1.0.101` of `generator-backend` (see [commit here](generator-backend) )

> TL;DR;
We need a new production deployment to get the fix
@joshuawilson is it done via Jenkins? do you know how to deploy to PROD?

**Comment by joshuawilson** [comment]
I'm not sure, I haven't published this before. @vpavlin do you know?

**Comment by qodfathr** [comment]
Fixed
